lib/pdf2img.js

```diff
@@ -47,8 +52,12 @@ Pdf2Img.prototype.convert = function(input, opts, callbackreturn) {
     opts = {};
   }
   var options = getOptions(opts);
+
+  // Set new timeout.
+  gm.prototype.options({timeout: options.timeout});
```

Comment:
Let's not set options onto the prototype, given that I just did some work to remove global state. Does chaining it not work? E.g. `gm(input).options({timeout: timeout}).whatever(...)`?

Reply:
good point, i'll try it.